src/app/seamly2d/dialogs/configpages/preferencesgraphicsviewpage.cpp

```diff
@@ -0,0 +1,162 @@
+/************************************************************************
+ **
+ **  @file   PreferencesGraphicsViewPage.cpp
+ **  @author DS Caskey
+ **  @date   10.17.2020
+ **
+ **  @brief
+ **  @copyright
+ **  This source code is part of the Valentine project, a pattern making
```

Comment:
This may have been auto-generated but you can remove 'Valentine" and replace with 'Seamly2D'